packs/github/actions/lib/base.py

```diff
@@ -16,11 +16,12 @@ def __init__(self, config):
         super(BaseGithubAction, self).__init__(config=config)
         token = self.config.get('token', None)
         token = token or None
-        self._client = Github(token)
+        base_url = self.config.get('base_url', None)
```

Comment:
Is that taken care of here? https://github.com/kevinSuttle/st2contrib/commit/8c1ff53e243119d4958f68336fc0da72fdd2d918